core/shared/src/main/scala/monocle/law/LensLaws.scala

```diff
@@ -14,12 +15,18 @@ class LensLaws[S, A](lens: Lens[S, A]) {
   def setGet(s: S, a: A): IsEq[A] =
     lens.get(lens.set(a)(s)) <==> a
 
-  def setIdempotent(s: S, a: A): IsEq[S] =
-    lens.set(a)(lens.set(a)(s)) <==> lens.set(a)(s)
```

Comment:
In fact, I think I am wrong. I don't think `setIdempotent` is redundant. I will add it back